src/Rule/AtLeast.php

```diff
@@ -27,23 +27,25 @@ final class AtLeast implements SerializableRuleInterface, SkipOnErrorInterface,
 
     public function __construct(
         /**
-         * The list of required attributes that will be checked.
+         * @var string[] The list of required attributes that will be checked.
          */
         private array $attributes,
         /**
-         * The minimum required quantity of filled attributes to pass the validation.
-         * Defaults to 1.
+         * @var int The minimum required quantity of filled attributes to pass the validation. Defaults to 1.
          */
         private int $min = 1,
+        private string $incorrectInputMessage = 'Value must be an array or an object.',
         /**
-         * Message to display in case of error.
+         * @var string Message to display in case of error.
          */
         private string $message = 'The model is not valid. Must have at least "{min}" filled attributes.',
-
         /**
          * @var bool|callable|null
          */
-        private $skipOnEmpty = null,
+        private mixed $skipOnEmpty = null,
+        /**
+         * @var bool
```

Comment:
Is this type-hint via phpdoc needed?

Reply:
Nope, it's a leftover after parameters' refactoring. Deleted it with couple of other similar ones.